components/builder-api/src/depot/server.rs

```diff
@@ -1462,7 +1462,14 @@ fn package_privacy_toggle(req: &mut Request) -> IronResult<Response> {
             opu.set_pkg(package);
 
             match route_message::<OriginPackageUpdate, NetOk>(req, &opu) {
-                Ok(_) => Ok(Response::with(status::Ok)),
+                Ok(_) => {
```

Comment:
thinking about this more - 1) I'm still not sure this logic is correct; and 2) I think we can actually eliminate this.  if we set a short expiration like 15 mins, things will just get naturally expired out, and we don't need to go hunting for all the channels that this package was in and manually try to delete them.  So net net, suggest we remove this section changes.

Reply:
A reasonable question to ask there is how the cache needs to behave in order to be effective in its purpose. If we can easily tolerate this query once every 15 minutes, perhaps the expiration time should be even lower.

Another thing to think about is whether many things will enter the cache right around the same time, and therefore leave the cache all at the same time, potentially leading to traffic spikes. If that's a concern, randomizing the TTL over an acceptable lifetime may be prudent.